src/plugins/plot/legend/PlotLegendItemCollapsed.vue

```diff
@@ -206,6 +206,9 @@ export default {
       const seriesIndexToRemove = this.seriesModels.findIndex(
         (series) => series.keyString === seriesToRemove.keyString
       );
+      if (seriesIndexToRemove === -1) {
+        return;
+      }
```

Comment:
don't remove a seriesModel if we don't find one in the list

Reply:
haha, that ... _makes sense_